### PR TITLE
Fixed function names of built-in scripts not showing up in debugger

### LINKED ISSUE #25198: Profiler does not show any function names for built-in scripts
**Godot version:**
v3.1-beta2

**OS/device including version:**
ArchLinux rolling + NVidia/Intel GPU

**Issue description:**
Profiler does not show names of functions that were called. Happens only for the built-in scripts.

Screenshot

**Steps to reproduce:**
1. Create a scene with a single `Node` object
2. Attach the following built-in script to the root node:

```gdscript
extends Node

func _process(delta: float) -> void:
    foo()

func foo():
    bar()

func bar():
    baz()

func baz():
    pass
```

If using an external script, the profiler works fine:

Screenshot

**Minimal reproduction project:**
Any project.

### PR DESCRIPTION
**Pull request by Piet-G**
Fixes #25198

### CODE REVIEW

File: editor/script_editor_debugger.cpp [hunk 1]
```diff
@@ -898,6 +898,10 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 					item.name = strings[2];
 					item.script = strings[0];
 					item.line = strings[1].to_int();
+				} else if (strings.size() == 4) {
```
Comment (akien-mga) [inline editor/script_editor_debugger.cpp]: A comment showing what the `== 3` and `== 4` cases refer to could be useful :)

Comment (DualMatrix) [reply editor/script_editor_debugger.cpp]: Added a comment explaining it

Comment (akien-mga) [reply editor/script_editor_debugger.cpp]: Seems like you pushed a version without the comment :)

Comment (DualMatrix) [reply editor/script_editor_debugger.cpp]: Oh yeah :/ Pushed

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**